iphone/Classes/TiUIPicker.m

```diff
@@ -225,6 +225,15 @@ - (void)setBackgroundColor_:(id)value
   }
 }
 
+- (void)setOverrideUserInterfaceStyle_:(id)args
+{
+  ENSURE_SINGLE_ARG(args, NSNumber);
+  if (picker != nil) {
+    int style = [TiUtils intValue:args def:UIUserInterfaceStyleUnspecified];
+    ((UIDatePicker *)[self picker]).overrideUserInterfaceStyle = style;
```

Comment:
I am very curious, why this manual change is necessary, as it should inherit it from `UIControl` internally.

Reply:
because you can't change the text color by hand and if you have a light mode app but a dark background somewhere it will look like the image in the description. Another image/client app is in https://tidev.slack.com/archives/C03CVQX2A/p1716034991145569. Having the option to change the interfaceStyle just for that element will at least show the text again as it will be white.

Reply:
Okay, thats an interesting edge case in the design spec. Then it's fine!